packages/pds/src/lexicon/types/tools/ozone/moderation/emitEvent.ts

```diff
@@ -26,6 +26,7 @@ export interface InputSchema {
     | ToolsOzoneModerationDefs.ModEventMuteReporter
     | ToolsOzoneModerationDefs.ModEventUnmuteReporter
     | ToolsOzoneModerationDefs.ModEventReverseTakedown
+    | ToolsOzoneModerationDefs.ModEventResolveAppeal
```

Comment:
This is unrelated but just happened to notice this because i was working on automod auto resolving and realized the generated types didn't include this so decided to just add it here.